.github/workflows/docker.yml

```diff
@@ -0,0 +1,34 @@
+name: ovn-docker-images
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+    paths: [ "dist/images/Dockerfile"]
```

Comment:
L7: should include glob_pattern since we have several Dockerfiles and the code before this PR matched all the Dockerfile

Reply:
ack. fixed (i think).
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
`*` should match zero or more characters.